tests/conftest.py

```diff
@@ -99,6 +99,15 @@ def pytest_addoption(parser):
     )
 
 
+def pytest_collection_modifyitems(config, items):
```

Comment:
Does this get hooked into things by virtue of the `pytest_` prefix?

Reply:
yes, during runtime it will collect all the items and mark them.

Reply:
I basically did what they had in the docs ;).
https://docs.pytest.org/en/latest/example/simple.html#control-skipping-of-tests-according-to-command-line-option